docs/detections/alerts-view-details.asciidoc

```diff
@@ -125,31 +173,81 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
 
 If these fields aren't available, {elastic-sec} does not perform the query and does not display threat intelligence data for the alert.
 
-Available threat indicator match data appears in the **Threat Match Detected** subsection.
+Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the *Threat Matched Detected* section under the *Threat intelligence* tab with additional threat match data.
 
-[role="screenshot"]
-image::images/matched-indicator-sub-sec.png[Threat Match Detected subsection with matched file hash, 600]
+Click **Threat intelligence** to expand the **Threat intelligence** sub-tab in the left panel. Within the sub-tab, matched threats are organized into two sections: <<threat-match-detected-section>> and <<enriched-with-threat-intelligence-section>>. Matched threats are ordered in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.
+
+[discrete]
+[[threat-match-detected-section]]
+==== Threat match detected
+
+The Threat match detected section is solely reserved for threat indicator matches identified by an indicator match rule. Threat indicator matches are produced whenever event data matches a threat indicator field value in your indicator index. 
+
+[discrete]
+[[enriched-with-threat-intelligence-section]]
+==== Enriched with threat intelligence
+
+The Enriched with threat intelligence section shows indicator matches that {elastic-sec} found when querying the alert for fields with threat intelligence. You can use the date time picker to modify the query time frame, which looks at the past 30 days by default. 
+
+Click the **Inspect** button, located on the far right of the threat label, to view more information on the query.
+
+NOTE: The event enrichment query uses the indices specified in the `securitySolution:defaultThreatIndex` advanced setting. For more information, refer to <<update-threat-intel-indices, Update default Elastic Security threat intelligence indices>>.
+
+[discrete]
+[[correlations-section]]
+=== Correlations 
 
-All other available threat intelligence data appears in the **Enriched with Threat Intelligence** subsection.
+The Correlations sub-section shows how an alert is related to other alerts and offers ways to investigate related alerts. Use this information to quickly find patterns between alerts and then take action. 
```

Comment:
same question as for the Threat Intelligence, sub-section could be better defined.

Maybe for each of these sections (Entities, Threat Intelligence, Correlations, Prevalence) you could have a small sentence explaining what we do in the overview (= what's in the right panel) then focus on the expanded section (= right panel which is the descriptions you currently have I believe)

Thoughts?

Reply:
Sorry, I'm not fully understanding the ask here. Are you requesting that every part of the left panel be documented? In the docs for each section within the right panel, I briefly touch on what's shown in the left panel expanded view, but I don't go into too much detail because the UI copy elaborates on the left panel components/tables/etc.

Reply:
Thinking about this further, I can create child sections within the main sections (Entities, Threat intelligence, Correlations, Prevalence) and move all information that's related to the the expanded view (i.e., what's shown in the left panel) and there. 

The TOC outline would look like this:

- [Insights](https://security-docs_3816.docs-preview.app.elstc.co/guide/en/security/master/view-alert-details.html#insights-section): Describe the purpose of the Insights section in the right panel. 
   - [Entities](https://security-docs_3816.docs-preview.app.elstc.co/guide/en/security/master/view-alert-details.html#entities-section): Describe the user and host overview that we provide in the right panel. 
     - Entity details: Describe the more detailed view that users get when they examine the Entities sub-tab in the left panel.
   - [Threat intelligence](https://security-docs_3816.docs-preview.app.elstc.co/guide/en/security/master/view-alert-details.html#threat-intelligence-section): Describe the threat intel overview that we provide in the right panel. 
      - Threat intelligence details: Describe the more detailed view that users get when they examine the Threat intelligence sub-tab in the left panel. Also describe how that view is populated.
   - [Correlations](https://security-docs_3816.docs-preview.app.elstc.co/guide/en/security/master/view-alert-details.html#correlations-section): Describe the summarized correlation details that we provide in the right panel. 
      - Correlation details: Describe what's shown in the individual tables that display in the Correlations sub-tab.
   - [Prevalence](https://security-docs_3816.docs-preview.app.elstc.co/guide/en/security/master/view-alert-details.html#prevalence-section): Describe what prevalence is and how it's calculated. 
      - Prevalence details: Explain how to use the table in the Correlations sub-tab.

Reply:
that's exactly what I was thinking yes. I like that for each bullet points above you have a sentence describing the functiontality in the right panel then a section where you provide more details about the left panel

Reply:
Ok cool. I'll rework the structure for the Insights main section. A lot of the explanatory material for the expanded view in the left panel already exists, so spinning up a new draft shouldn't take too long.

Reply:
Resolving in next batch of updates.